src/scoring_functions.py

```diff
@@ -28,4 +33,40 @@ def entropy(predicted_user_profiles, predicted_item_attributes):
     entropy = - predicted_scores * np.log(predicted_scores + globals.EPS)
     
     re_ranked_scores = predicted_scores + alpha * entropy
-    return re_ranked_scores
\ No newline at end of file
+    assert (re_ranked_scores >= 0).all(), "Some scores are negative."
+
+    return re_ranked_scores
+
+
+def content_fairness(predicted_user_profiles, predicted_item_attributes):
+    slate_size = 10
```

Comment:
if there's any chance that we reduce/change the slate size maybe consider making this a hyperparameter

Reply:
Unfortunately, you cannot pass parameters to the score_fn. It is obviously bad practice but I would say it's totally fine for our project.